sqlite/ext/misc/completion.c

```diff
@@ -332,6 +339,10 @@ static int completionNext(sqlite3_vtab_cursor *cur){
           pCur->szRow = strlen(pCur->zCurrentRow);
         }
         iCol = -1;
+         */
```

Comment:
[Optional] I'd remove instead of commenting them out.

Reply:
creating a new ifdef -- it is good to have how to add custom list for future reference.